src/main/java/com/revature/services/UserService.java

```diff
@@ -24,20 +28,42 @@
 	 */
 
 	public User getUserById(int id) {
-		Optional<User> optUser = userRepository.findById(id);
-		if(optUser.isPresent()) {
-			User user = optUser.get();
 			try {
+				Optional<User> optUser = userRepository.findById(id);
+				if(optUser.isPresent()) {
+					User user = optUser.get();
 				
 				user.setPoints(rssService.getPoints(id));
-			}catch(Exception e){
-				System.out.println(e.getLocalizedMessage());
+				return userRepository.save(user);
+				}
+			}catch(Exception e) {
+				//add logger
```

Comment:
Add the actual logging output - otherwise we're allowing exceptions to occur silently.

Reply:
added